.github/workflows/clang-tidy.yml

```diff
@@ -17,5 +17,8 @@ jobs:
       with:
         token: ${{secrets.ACCESS_TOKEN}}
 
+    - name: install dependencies
+      run: pip3 install dataclasses
```

Comment:
Where is this required?

Reply:
IMO worth adding a comment in the file or commit message, so we can remove it later on.